setup.py

```diff
@@ -204,9 +204,9 @@ def op_enabled(op_name):
 git_branch_cmd = "git rev-parse --abbrev-ref HEAD"
 if command_exists('git') and not is_env_set('DS_BUILD_STRING'):
     try:
-        result = subprocess.check_output(git_hash_cmd, shell=True)
```

Comment:
These instances are required in order to build deepspeed and get the git commit.